From b709f1b627ca0c9deed69209a27b809d599e3305 Mon Sep 17 00:00:00 2001
From: Pierre-Louis Bossart <pierre-louis.bossart@linux.intel.com>
Date: Tue, 3 Jul 2018 12:33:03 -0500
Subject: [PATCH 1757/2328] ASoC: Intel: fix Kconfig typos

Fix ASoC: Intel: select relevant machine drivers for SOF
s/INTEL_SKL/INTEL_SKYLAKE

Signed-off-by: Pierre-Louis Bossart <pierre-louis.bossart@linux.intel.com>
---
 sound/soc/intel/boards/Kconfig | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/sound/soc/intel/boards/Kconfig b/sound/soc/intel/boards/Kconfig
index 67ca087..1f53302 100644
--- a/sound/soc/intel/boards/Kconfig
+++ b/sound/soc/intel/boards/Kconfig
@@ -209,9 +209,9 @@ config SND_SOC_INTEL_SKL_NAU88L25_MAX98357A_MACH
 	  Say Y or m if you have such a device. This is a recommended option.
 	  If unsure select "N".
 
-endif ## SND_SOC_INTEL_SKL
+endif ## SND_SOC_INTEL_SKYLAKE
 
-if SND_SOC_INTEL_SKL || SND_SOC_SOF_APOLLOLAKE
+if SND_SOC_INTEL_SKYLAKE || SND_SOC_SOF_APOLLOLAKE
 
 config SND_SOC_INTEL_BXT_DA7219_MAX98357A_MACH
 	tristate "Broxton with DA7219 and MAX98357A in I2S Mode"
@@ -259,9 +259,9 @@ config SND_SOC_INTEL_BXT_TDF8532_MACH
 	  Say Y if you have such a device.
 	  If unsure select "N".
 
-endif ##  SND_SOC_INTEL_SKL || SND_SOC_SOF_APOLLOLAKE
+endif ##  SND_SOC_INTEL_SKYLAKE || SND_SOC_SOF_APOLLOLAKE
 
-if  SND_SOC_INTEL_SKL
+if  SND_SOC_INTEL_SKYLAKE
 
 config SND_SOC_INTEL_KBL_RT5663_MAX98927_MACH
 	tristate "KBL with RT5663 and MAX98927 in I2S Mode"
-- 
2.7.4

